Documents/Development/Contributing-Guidelines.md

```diff
@@ -1,3 +1,80 @@
 # Contributing Guidelines for Krypton Development
 
-BEFORE CONTRIBUTING please read
\ No newline at end of file
+BEFORE CONTRIBUTING please read _all_ of the rules below are guidelines, which means that they should be followed when possible. Please do not take them literally.
+
+## Discussions
+
+If you wish to discuss about the development of the toolkit, please use the areas as provided in either in the [Standard Toolkit](https://github.com/Krypton-Suite/Standard-Toolkit/discussions) or the [Extended Toolkit](https://github.com/Krypton-Suite/Extended-Toolkit/discussions).
+
+## Rules
+
+The developers **respects** people, despite their race, gender, religion, height or culture. Any user who posts offensive or disrespectful content regarding race, gender, religion, height or culture **will be immediately banned from this repository and Discord by extension**. No exception will be made.
```

Comment:
Pluralisation on both - one of them needs to be removed

Reply:
Done